### PR TITLE
Prep and migrate

### PR DESCRIPTION
**Pull request by jburel**
* Add the commit from 5.6.1
* include changes from autogen

### CODE REVIEW

File: omero/users/history.rst [hunk 1]
```diff
@@ -26,12 +26,46 @@ omero clients:
 - omero-web :web_source:`CHANGELOG.md<CHANGELOG.md>`
 
 .. Content for this page should be opened as a PR against
-.. https://github.com/ome/openmicroscopy/blob/develop/history.rst
+.. https://github.com/openmicroscopy/openmicroscopy/blob/develop/history.rst
```
Comment (mtbc) [inline omero/users/history.rst]: This change doesn't seem right. (Also below.)

Comment (jburel) [reply omero/users/history.rst]: Maybe from the template file. I will need to check that

File: omero/sysadmins/config.rst [hunk 2]
```diff
@@ -1138,10 +1138,30 @@ Default: `60`
 Performance
 -----------
 
+.. property:: omero.sessions.max_user_time_to_idle
+
+omero.sessions.max_user_time_to_idle
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Sets the maximum duration a user can request before a login
+is required due to inactivity.
```
Comment (will-moore) [inline omero/sysadmins/config.rst:1146]: In seconds? Milliseconds?

File: omero/sysadmins/config.rst [hunk 3]
```diff
@@ -1138,10 +1138,30 @@ Default: `60`
...
+
+Default: `6000000`
+
+.. property:: omero.sessions.max_user_time_to_live
+
+omero.sessions.max_user_time_to_live
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Sets the maximum duration a user can request before a login
+is required (0 signifies never).
```
Comment (will-moore) [inline omero/sysadmins/config.rst:1155]: In seconds? Milliseconds?

File: omero/sysadmins/config.rst (hunk 2 above)
Comment (jburel) [reply omero/sysadmins/config.rst:1146]: That will have to be fixed in openmicroscopy

**Comment by will-moore** [comment]
looks good 👍

**Pull request merged**